src/ILCompiler.WebAssembly/src/Compiler/WebAssemblyMetadataBlockingPolicy.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Internal.TypeSystem;
+
+namespace ILCompiler.Compiler
```

Comment:
Could you make this `namespace ILCompiler`? We don't use `ILCompiler.Compiler` as a namespace.